src/AutorestSwift/Code Generator/Logger.swift

```diff
@@ -67,6 +67,8 @@ class FileLogger {
 
     let url: URL
 
+    static let instance = FileLogger(withFileName: "autorest-swift-debug.log")
```

Comment:
Typically in iOS lingo this would be called `shared`.

Reply:
Renamed.